### DISCUSSION TITLE
Semantic editing packages

### DISCUSSION THREAD

**Discussion by ethan-leba** [emacs-tree-sitter/elisp-tree-sitter#135] [Q&A] [answered]
Hi all,

I've been thinking about creating a package like [lispy](https://github.com/abo-abo/lispy) for non-lisp languages using `emacs-tree-sitter`. I wasn't able to find any packages of the sort from a quick search, but I wanted to ask here if something like this already exists (if so, great!) so that I'm not reinventing the wheel.

**Comment by elzorrorebelde** [comment, marked as answer]
AFAIK there is nothing like that for non-lisp programming languages
